easybuild/easyconfigs/p/Pango/Pango-1.40.1-intel-2016a.eb

```diff
@@ -34,7 +34,14 @@ builddependencies = [
 
 configopts = "--disable-silent-rules --enable-introspection=yes --enable-static --enable-shared "
 
+sanity_check_paths = {
+    'files': ['bin/pango-view', 'lib/libpango-1.0.%s' % SHLIB_EXT, 'lib/libpangocairo-1.0.%s' % SHLIB_EXT,
+              'lib/libpangoft2-1.0.%s' % SHLIB_EXT, 'lib/libpangoxft-1.0.%s' % SHLIB_EXT],
```

Comment:
@akesandgren Same here, no `lib/libpangoxft-1.0.so` in `Pango` 1.40.1, so please drop last entry?